pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -224,11 +302,21 @@ func getHugepagesAllocationUnitOptions(hugepagesSize string, hugepagesCount int3
 	}
 }
 
-func addFile(ignitionConfig *igntypes.Config, src string, dst string, mode *int) error {
-	content, err := ioutil.ReadFile(src)
-	if err != nil {
-		return err
+func getRpsUnitOptions(rpsMask string) []*unit.UnitOption {
```

Comment:
nit: use upper case letters for acronyms - getRPSUnitOptions

Reply:
will change, thanks